Frameworks/CoreGraphics/CGPath.mm

```diff
@@ -222,9 +222,9 @@ void _delayedEndFigureCall() {
     // closed.
     bool m_hasGeometryStarted;
     // True if this geometry can end its own figures.
-    bool m_allowsFigureCalls;
+    int m_allowsFigureCalls;
```

Comment:
Yes, saw that later on.  but this is not technically clean-up, it is a design change, would be useful to call that out in the comments.

---
In reply to: [119761248](https://github.com/Microsoft/WinObjC/pull/2753#discussion_r119761248) [](ancestors = 119761248)